ui/desktop/app/helpers/time-remaining.js

```diff
@@ -0,0 +1,44 @@
+import Helper from '@ember/component/helper';
```

Comment:
@calcaide @cameronperera should this helper be located in `core` or is it ok in the `ui/desktop` location?

Reply:
This is true but we've also had quite a few date/time formatters already in core only used in one UI so we should be consistent

Reply:
I do agree with @ZedLi 

It is true that this helper, right now is use JUST on desktop. But a time-remaning helper, if we do it pure, so it receives 2 params, and returns the subtract of them, can be reusable everywhere.

Also, @ZedLi  mentions we have time related helpers within core, even tho we just use them in Admin.

I second @ZedLi  comment, for consistency’s sake we should have it within core.

Reply:
I agree with that reasoning. It should be in core with the other time related helpers. Thanks!

Reply:
Moved the time-remaining helper to `addons/core`